packages/app/cypress/e2e/runner/retries.mochaEvents.snapshots.ts

```diff
@@ -0,0 +1,7120 @@
+export const snapshots = {
```

Comment:
Copy paste from the snapshots previously existing in `packages/runner/__snapshots__`.